Assets/MRTK/SDK/Features/DataBinding/Scripts/Interfaces/IDataObjectPool.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Data
+{
+
+    /// <summary>
+    /// A data pool of objects that can be used for object re-use.
+    /// 
+    /// This is designed to reduce memory allocations and hence
+    /// the frequency and duration of garbage collections. It can
+    /// also reduce the instantiation time of a prefab when populating
+    /// large lists.
+    /// </summary>
+    /// 
+    public interface IDataObjectPool
+    {
+
+        void SetMaximumPoolSize(int maxSize, bool resizeNow);
+
```

Comment:
More whitespace; many other files have a lot of extra whitespace too, but I don't want to spam comments. Reducing unnecessary whitespace is great for readability!